pkg/registry/debian/debian.go

```diff
@@ -266,9 +269,88 @@ func (r HTTPRegistry) DSC(ctx context.Context, component, name, version string)
 	return DSCURI, d, err
 }
 
+// fileInfo is the response from the binfiles endpoint on the snapshot service.
+type fileInfo struct {
+	// FileInfo is a map from file hash to extra info
+	FileInfo map[string][]struct {
+		Name string
+	}
+	// Result is a list of file hashes with architecture
```

Comment:
this comment describes the structure instead of the semantics. if we're not adding information, let's remove the comment